src/data_formatter.ts

```diff
@@ -7,9 +7,6 @@ export default class DataFormatter {
 
   setValues(data) {
     if (this.ctrl.series && this.ctrl.series.length > 0) {
-      let highestValue = 0;
```

Comment:
Can I get some reasoning why did you remove the calculation of the `lowestValue`, `highestValue` and `valueRange` from this file?

Reply:
Well, this calculation was done when the data was loaded and, if we filtered on the "map side", those values would still be the same. You see, as they've built this plugin, there's two important "events", data load and screen load. Those calculations were happening on the data load (only when you load the scree or reload the data). Now, as we're changing how the circle's sizes are calculated with the data loaded, we would need to reload the data every time we changed the configurations. Now, this calculation is on the screen load(or reload). If the configuration changes, we calc it again (with the new values) and show them again.
Also, this formatter has a lot of duplicated and messy code. My intentions here are to clean up this file as best as I can, remove some duplications and make it more simple. Removing this from here helped this a lot.
If you look into Worldmap.ts:147, you gonna see how I did it waaaaaaaaaaay simpler, using Math.min and Math.max.